packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -130,8 +132,17 @@ abstract class MobileAd {
   ///
   /// The [listener] will be notified when the ad has finished loading or fails
   /// to do so. An ad that fails to load will not be shown.
-  Future<bool> show() {
-    return _channel.invokeMethod("showAd", <String, dynamic>{'id': id});
+  ///
+  /// anchorOffset is the logical pixel offset from the edge of the screen (default 0.0)
+  /// anchorType place advert at top (anchorTop) or bottom (anchorBottom) of screen (default anchorBottom)
+  Future<bool> show(
+      {double anchorOffset = 0.0,
+      AnchorType anchorType = AnchorType.anchorBottom}) {
+    return _channel.invokeMethod("showAd", <String, dynamic>{
+      'id': id,
+      'anchorOffset': anchorOffset.toString(),
+      'anchorType': anchorType.index.toString()
```

Comment:
No need to send a string representation of the index here. Either just the index or, alternatively, a human-readable string representing the intention, e.g. "bottom".